evalml/tests/model_understanding_tests/prediction_explanations_tests/test_explainers.py

```diff
@@ -393,23 +393,36 @@ def test_output_format_checked():
 """.format(multiclass_table=multiclass_table)
 
 
-@pytest.mark.parametrize("problem_type,output_format,answer,explain_predictions_answer",
-                         [(ProblemTypes.REGRESSION, "text", regression_best_worst_answer, no_best_worst_answer),
-                          (ProblemTypes.REGRESSION, "dict", regression_best_worst_answer_dict, no_best_worst_answer_dict),
-                          (ProblemTypes.BINARY, "text", binary_best_worst_answer, no_best_worst_answer),
-                          (ProblemTypes.BINARY, "dict", binary_best_worst_answer_dict, no_best_worst_answer_dict),
-                          (ProblemTypes.MULTICLASS, "text", multiclass_best_worst_answer, multiclass_no_best_worst_answer),
-                          (ProblemTypes.MULTICLASS, "dict", multiclass_best_worst_answer_dict, no_best_worst_answer_dict)])
+@pytest.mark.parametrize("problem_type,output_format,answer,explain_predictions_answer,custom_index",
+                         [(ProblemTypes.REGRESSION, "text", regression_best_worst_answer, no_best_worst_answer, [0, 1]),
+                          (ProblemTypes.REGRESSION, "dict", regression_best_worst_answer_dict, no_best_worst_answer_dict, [0, 1]),
+                          (ProblemTypes.REGRESSION, "text", regression_best_worst_answer, no_best_worst_answer, [4, 23]),
+                          (ProblemTypes.REGRESSION, "dict", regression_best_worst_answer_dict, no_best_worst_answer_dict, [4, 10]),
+                          (ProblemTypes.REGRESSION, "text", regression_best_worst_answer, no_best_worst_answer, ["foo", "bar"]),
```

Comment:
Wow I had no idea indices could be set to anything that wasn't numbers. This is really cool!